bitnami/logstash/templates/headless-svc.yaml

```
@@ -20,5 +20,10 @@ spec:
     {{- range $port := .Values.service.ports }}
     - {{- omit (include "common.tplvalues.render" ( dict "value" $port "context" $ ) | fromYaml) "nodePort" | toYaml | nindent 6 }}
     {{- end }}
+    {{- if .Values.service.extraPorts }}
+    {{- range $port := .Values.service.extraPorts }}
+    - {{- omit (include "common.tplvalues.render" ( dict "value" $port "context" $ ) | fromYaml) "nodePort" | toYaml | nindent 6 }}
+    {{- end }}
```

WDYT about using this (as other solutions)?

```suggestion
    {{- include "common.tplvalues.render" (dict "value" .Values.service.extraPorts "context" $) | nindent 6 }}
```

---

Should work for me but it might cause an issue when someone adds a `nodePort` to `extraPorts`. That's why I handled the `extraPorts` just like the `ports`.